incubator/swift/image/Dockerfile-stack

```diff
@@ -1,9 +1,4 @@
-FROM swift:5.0
-
-RUN apt-get update \
- && apt-get install -y zlib1g-dev libcurl4-openssl-dev libssl-dev \
- && apt-get clean \
- && echo 'Finished installing dependencies'
+FROM swift:5.1
```

Comment:
We should leave `zlib1g-dev` in for now as I think Chris's blog post needs it for the Vapor demo.

Reply:
https://medium.com/appsody/server-side-swift-a77bff1d8f1

Reply:
Could the blog post use the `kitura` stack instead for the Kitura bit? That would seem more appropriate.

We can certainly add zlib back in again - turns out things like `async-http-client` now require that too.

Reply:
Yes, leaving zlib in here and updating the Kitura bit of the blog post is my preference.

Reply:
We've decided to leave those dependencies in the swift stack for now, and a better long-term solution would be to discover them from the dependency tree.  Can discuss in https://github.com/appsody/stacks/issues/432